client/src/components/Ticketing/ticketingmanager/Refund/RefundOrderTicket.tsx

```diff
@@ -1,42 +1,174 @@
+                      <path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd" />
+                    </svg>
+                  </div>
+                </button>
+              </div>
+            </label>
+          </div>
+          <div className='w-full min-w-min'>
+            <div className='grid grid-cols-5 gap-2 bg-gray-200 h-18 rounded-lg shadow-md px-2 mb-2 font-bold'>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                  Name
+              </div>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                <div className="border-l border-l-gray-500/50 pl-2">
+                  Date
+                </div>
+              </div>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                <div className="border-l border-l-gray-500/50 pl-2">
+                  Showing(s)
+                </div>
+              </div>
+              <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                <div className="border-l border-l-gray-500/50 pl-2">
+                  Total
+                </div>
+              </div>
+              <div className='row-start-1 justify-self-center py-2 col-span-1'></div>
             </div>
-            <button className='bg-red-700
-              hover:bg-red-500 text-white mt-5 py-2 px-4 rounded'>
-              Refund
-            </button>
+            {mappedInstances.length === 0 ? (
+              <div className="text-center text-gray-600">No current results</div>
+            ) : (
+              mappedInstances.map((instance, index) => (
+                <div key={index} className='grid grid-cols-5 gap-2 bg-gray-200 rounded-lg shadow-md px-2 mb-2 hover:bg-gray-300'>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {instance.name}
+                  </div>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {instance.date}
+                  </div>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {Array.isArray(instance.showings) ? (
+                      instance.showings.map((showing, showingIndex) => (
+                        <div key={showingIndex}>{showing}</div>
+                      ))
+                    ) : (
+                      <div>{instance.showings}</div>
+                    )}
+                  </div>
+                  <div className='row-start-1 justify-self-start pl-2 py-2 col-span-1'>
+                    {instance.price}
+                  </div>
+                  <div className='row-start-1 justify-self-start py-2 col-span-1'>
+                    <button
+                      onClick={async () => await onSubmit(instance.orderID)}
```

Comment:
Async and await can be used here but it's not required because you declared the `onSubmit` function as async on line 17. So if you change this to just a regular arrow function like `() => onSubmit(instance.orderID)`, it'll work as intended as well. The `onSubmit` declared on line 17 will return a promise so it makes sense that would would do await onSubmit(...) here, but since you wrapped that in an async function, it returns another promise which isn't being used. So you can kinda skip the middle step and and remove the async await on this line. I hope that makes sense!